src/middlewared/middlewared/test/integration/utils/client.py

```diff
@@ -215,8 +215,9 @@ def host():
     return truenas_server
 
 
-def host_websocket_uri(host_ip=None):
-    return f"ws://{host_ip or host().ip}/api/current"
+def host_websocket_uri(host_ip=None, ssl=False):
```

Comment:
Would it be better to have `ssl=True` as the default?

Reply:
Well, we are defaulting to `ssl=True` for `Client()` connections.

Reply:
But sure, why not :)